drivers/wifi/Kconfig

```diff
@@ -35,6 +35,13 @@ config WIFI_USE_NATIVE_NETWORKING
 	  When selected, this hidden configuration enables Wi-Fi driver
 	  to use native ethernet stack interface.
 
+config BUILD_ONLY_NO_BLOBS
```

Comment:
I guess `drivers/wifi/Kconfig` is the wrong place for this, now that it's completely generic. Perhaps `modules/Kconfig` instead?

Reply:
`BUILD_WITHOUT_BLOBS` is fine

Reply:
`BUILD_WITHOUT_BLOBS` is fine, `ONLY` was added to imply that this is `build-only` and cannot be used on a real board for testing.

Reply:
> `BUILD_WITHOUT_BLOBS` is fine, `ONLY` was added to imply that this is `build-only` and cannot be used on a real board for testing.

Yeah, that was my idea when I proposed it, but then I realized that when you try to read it as normal English it's mildly confusing. Either way is fine with me.

Reply:
Also, as long as we output a clear warning during build, I think that should do the job of informing the user that the result is something non-functional.